uid-pages/page-admin-profile-list/src/adminProfileListBonitaV1.json

```diff
@@ -1122,8 +1122,8 @@
                               "value" : ""
                             },
                             "hidden" : {
-                              "type" : "constant",
-                              "value" : false
+                              "type" : "expression",
+                              "value" : "!featuresListCtrl.isFeatureAvailable(\"WEB_PROFILE_PO\")"
```

Comment:
I think export can be done on provided profiles as well (to export the mapping). no need for feature check in this case